src/dmd/func.d

```diff
@@ -2436,16 +2436,44 @@ extern (C++) int overloadApply(Dsymbol fstart, void* param, int function(void*,
     return overloadApply(fstart, s => (*fp)(param, s));
 }
 
-void MODMatchToBuffer(OutBuffer* buf, ubyte lhsMod, ubyte rhsMod)
+/**
+Checks for mismatching modifiers between `lhsMod` and `rhsMod` and prints the
+mismatching modifiers to `buf`.
+
+The modifiers of the `lhsMod` mismatching the ones with the `rhsMod` are printed, i.e.
+lhs(shared) vs. rhs() prints "`shared`", wheras lhs() vs rhs(shared) prints "non-shared".
+
+Params:
+    buf = output buffer to write to
+    lhsMod = modifier on the left-hand side
+    lhsMod = modifier on the right-hand side
+
+Returns:
+
+A tuple with `isMutable` and `isNotShared` set
+if the `lhsMod` is missing those modifiers (compared to rhs).
+*/
+auto MODMatchToBuffer(OutBuffer* buf, ubyte lhsMod, ubyte rhsMod)
```

Comment:
`ref OutBuffer` instead of `OutBuffer*`?

Reply:
One change at a time?
(This is an existing function)

Reply:
Ah, I overlooked it was existing.